default_ocp_versions.json

```diff
@@ -1 +0,0 @@
-./data/default_ocp_versions.json
```

Comment:
Is it okay that this was removed?

Reply:
Added back. It was removed accidentally on rebase.